src/sentry/tasks/post_process.py

```diff
@@ -1056,6 +1056,28 @@ def fire_error_processed(job: PostProcessJob):
         )
 
 
+def sdk_crash_monitoring(job: PostProcessJob):
+    from sentry.utils.sdk_crashes.sdk_crash_detection import sdk_crash_detection
+
+    if job["is_reprocessed"]:
+        return
+
+    event = job["event"]
+
+    if not features.has("organizations:sdk-crash-detection", event.project.organization):
+        return
```

Comment:
nit: should we move this to the top of the function?

Reply:
@iker The `job["is_reprocessed"]` check is cheaper than the feature check, so I would keep this as-is.